### PR TITLE
Update zha.markdown with more detailed use-case description

### PR DESCRIPTION
**Pull request by Hedda**
**Description:**

Update zha.markdown with more detailed use-case description about its HA native functionality.

This fleshing out of the description is to try to better explain to end-users (as well as to developers) that from an end-user point-of-view this Zigpy integration adds a native Zigbee function implementation to HA, (similar to what the OpenZWave integration does for Z-Wave in HA). I think that this or a similar fleshed out description is needed so that end-users that are new to HA will know the difference between this integration as others that use third-party hardware or software application acting as a gateway or hub, such as for example DeConz, Zigbee2mqtt, Philips Hue Hub, Ikea Tradfri Gateway, and Samsung SmartThings Hub.

Also added a link to ZiGate website and clarified its firmware version requirement.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Zigbee Home Automation"
-description: "Instructions on how to integrate your Zigbee Home Automation (ZHA) devices within Home Assistant."
+description: "Instructions on how to integrate your Zigbee Home Automation (ZHA) devices within Home Assistant using compatible USB sticks or modules hardware."
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
description: "Instructions on how to integrate your Zigbee Home Automation (ZHA) devices within Home Assistant using compatible USB sticks or modules hardware."
```

A title should be short and the point... Please revert.

File: source/_integrations/zha.markdown [hunk 2]
```diff
@@ -51,7 +55,7 @@ The custom quirks implementations for zigpy implemented as ZHA Device Handlers f
   - [ConBee USB adapter from Dresden-Elektronik](https://www.dresden-elektronik.de/conbee/)
   - [RaspBee Raspberry Pi Shield from Dresden-Elektronik](https://www.dresden-elektronik.de/raspbee/)
 - ZiGate based radios (via the [zigpy-zigate](https://github.com/doudz/zigpy-zigate) library for zigpy)
-  - ZiGate USB modules (require firmware 3.1a or later)
+  - [ZiGate USB modules](https://zigate.fr/) (Note! first requires that ZiGate firmware has been updated to 3.1a or later)
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
  - [ZiGate USB modules](https://zigate.fr/) (Requires that the ZiGate firmware has been updated to 3.1a or later)
```

File: source/_integrations/zha.markdown (hunk 1 above)
Comment (Hedda) [reply source/_integrations/zha.markdown]: Hope that it is OK with; "Instructions for native integration of Zigbee Home Automation (ZHA) devices in Home Assistant.".

File: source/_integrations/zha.markdown [hunk 3]
```diff
@@ -16,7 +16,11 @@ featured: true
 ---
 
 [Zigbee Home Automation](http://www.zigbee.org/zigbee-for-developers/applicationstandards/zigbeehomeautomation/)
-integration for Home Assistant allows you to connect many off-the-shelf Zigbee based devices to Home Assistant, using one of the available Zigbee radio modules compatible with [zigpy](https://github.com/zigpy/zigpy) (an open source Python library implementing a Zigbee stack, which in turn relies on separate libraries which can each interface a with Zigbee radio module a different manufacturer).
+integration for Home Assistant allows you to connect many off-the-shelf Zigbee based devices directly to Home Assistant using one of the several available Zigbee radio modules hardware compatible with [zigpy](https://github.com/zigpy/zigpy). 
+
+Zigpy is an open source Python library implementing a Zigbee stack, it functions as a middleware which in turn relies on separate Python libraries which each acts a wrapper to abstract the unique interfaces of Zigbee radio modules from different manufacturer).
```
Comment (frenck) [inline source/_integrations/zha.markdown:21]: Can we rewrite this in a way a normal user could understand this better?
This sentence is pretty long and full of technical words, most users would have a hard time reading this.

Comment (Hedda) [reply source/_integrations/zha.markdown:21]: Suggestions on better phrasing? My original thinking was that most Home Assistant users today are very technical and that the introduction text is also for new developers too, not just end-users.

Comment (klaasnicolaas) [reply source/_integrations/zha.markdown:21]: Developers have their own website, with the doc website the end users are always central and especially people who start at 0

**Comment by frenck** [comment]
This PR is running stale, no progress has been made in the past month.
Closing up this PR, feel free to re-open when ready.

Thanks 👍

**Pull request closed without merging**